main.rkt

```diff
@@ -22,6 +25,9 @@
 
 ;; =============================================================================
 
+(define (square n)
+  (expt n 2))
+
```

Comment:
Amazing that this isn't here. For the sake of all the power series stuff, we should add

```racket
(define (cube n)
  (expt n 3))
```

In my file I also defined some useful constants

```racket
(define π pi)
(define π/2 (/ π 2))
(define π/3 (/ π 3))
(define π/4 (/ π 4))
(define π/6 (/ π 6))
(define π/12 (/ π 12))
(define 2/π (/ 2 π))
(define 3π/4 (* 3 π/4))
(define e (exp 0))
(define 1/e (/ 1 e))
```
These are probably best added here for now, then when we amass enough of them, we should export a `constants.rkt` and just import from that.